Remove override style for pinned column's text (#2756)
jpmorganchase/salt-ds: .changeset/chilly-cherries-build.md

---
"@salt-ds/ag-grid-theme": patch
---

> Remove override style for pinned column's text

Reviewer: Can you put the reasoning behind this was done in the changeset please, as well as make the changeset past tense.

---
"@salt-ds/ag-grid-theme": patch
---

> Removed override style for pinned column's text.
> Bug fix: The cell text from the pinned column was overlapping the cell next to it.

Reviewer: ```suggestion
Cell text from a pinned column no longer overlaps the cell next to it.
```
